src/controller/jobmonitor/monitor.go

```diff
@@ -70,6 +70,7 @@ type MonitorController interface {
 	PauseJobQueues(ctx context.Context, jobType string) error
 	// ResumeJobQueues resume the job queue by type
 	ResumeJobQueues(ctx context.Context, jobType string) error
+	JobLog(ctx context.Context, jobID string) ([]byte, error)
```

Comment:
Follow the style of function name `Action + Object` as above, `GetJobLog` seems better.

Reply:
Updated